search/lib/api/wfs.js

```diff
@@ -144,7 +144,7 @@ async function getGranules (request, response) {
   try {
     const cmrParams = Object.assign(
       { provider: providerId },
-      cmr.convertParams(cmr.STAC_SEARCH_PARAMS_CONVERSION_MAP, query)
+      await cmr.convertParams(providerId, query)
```

Comment:
Is this ticket supposed to fix the case when you pass multiple collections in the search like /search?collections=name1.v1,name2.v2,name3.v3?

Reply:
Yes, this supports multiple collections passed in via search, but only for POST.
There's a more general bug with GET requests, see CMR-7140